### ISSUE TITLE
[BE & FE] Display notification when OTP used

### ISSUE THREAD

**Issue by minatisleeping** [PiedTeam/NikeCloneTraining-BE-Project#136]
BE validation case input OTP used and continue to submit. Output: return message error
FE show message error message
User cầm cái OTP đã sử dụng rồi để dùng lại, mình chưa check case này
Nhật: bắt lỗi này quăng cho fe biết
Tiến: nhận lỗi từ Nhật rồi throw ra UI cho user biết

**Comment by minatisleeping** [comment]
**Description**: When user want to veriry their account, system send to them OTP based on email_phone. In this contex, I stimulate account haven't verified their account yet..
### I put 2 middleware in here: 
- The first I checked if user have already verified account I throw err with message: "Your account is already verified!" like video below..

https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/0bd0eceb-25b4-4bc8-b75f-3130e46d5407

- The second I checked if user input OTP used before, unfortunately, system not updated that account to status: 1(verified), I will throw error with message: "OTP has been already used or has been expired", OTP in vid below is used(status: 0[Unavailable])

https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/3005c90c-3953-4577-ad15-897d7ea52016

**In summary:** It really **difficult** to happen in case2 as in case1 if user have verified, I will throw error. However if xui xui, user passed case1, I still catch case2..
